frontend/packages/dev-console/integration-tests/tests/serverless.scenario.ts

```diff
@@ -0,0 +1,148 @@
+import { execSync } from 'child_process';
+import { protractor, by, browser, ExpectedConditions as until } from 'protractor';
+import {
+  appHost,
+  checkLogs,
+  checkErrors,
+  testName,
+  retry,
+} from '../../../../integration-tests/protractor.conf';
+import {
+  installServerlessOperator,
+  firstKebabMenu,
+  machineSetsInput,
+  submitCount,
+  servicesValidation,
+  serverlessNavItemValidation,
+  cellClass,
+  knativeServingNS,
+  pageSidebar,
+} from '../views/serverless.view';
+import { switchPerspective, Perspective, sideHeader } from '../views/dev-perspective.view';
+import * as sidenavView from '../../../../integration-tests/views/sidenav.view';
+import * as crudView from '../../../../integration-tests/views/crud.view';
+import * as catalogView from '../../../../integration-tests/views/catalog.view';
+import * as catalogPageView from '../../../../integration-tests/views/catalog-page.view';
+import * as operatorView from '../../../operator-lifecycle-manager/integration-tests/views/operator.view';
+import * as operatorHubView from '../../../operator-lifecycle-manager/integration-tests/views/operator-hub.view';
+
+const JASMINE_DEFAULT_TIMEOUT_INTERVAL = jasmine.DEFAULT_TIMEOUT_INTERVAL;
+const JASMINE_EXTENDED_TIMEOUT_INTERVAL = 3000 * 60 * 5;
+
+describe('Serverless', async () => {
+  beforeAll(async () => {
+    await browser.get(`${appHost}/k8s/cluster/projects`);
+    await switchPerspective(Perspective.Administrator);
+    expect(sideHeader.getText()).toContain('Administrator');
+
+    // Extend the default jasmine timeout interval just in case it takes a while for the htpasswd idp to be ready
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = JASMINE_EXTENDED_TIMEOUT_INTERVAL;
+  });
+
+  afterAll(() => {
+    // Set jasmine timeout interval back to the original value after these tests are done
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = JASMINE_DEFAULT_TIMEOUT_INTERVAL;
+  });
+
+  afterEach(() => {
+    checkLogs();
+    checkErrors();
+  });
+
+  it('displays subscription creation form for selected Operator', async () => {
```

Comment:
nit: this scenario is for serverless so we can be more specific `displays subscription creation form for serverless Operator`

Reply:
Okay. I will update the it block.